src/content/docs/fr/guides/integrations-guide/cloudflare.mdx

```diff
@@ -0,0 +1,402 @@
+---
+type: integration
+title: '@astrojs/cloudflare'
+description:  Apprendre à utiliser l'adaptateur SSR @astrojs/cloudflare pour déployer votre projet Astro.
+githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/cloudflare/'
+hasREADME: true
+category: adapter
+i18nReady: true
+---
+
+import Video from '~/components/Video.astro';
+import DontEditWarning from '~/components/DontEditWarning.astro';
+
+<DontEditWarning/>
+
+Un adaptateur SSR pour utiliser les fonctions de Cloudflare Pages. Ecrivez votre code en Astro/Javascript et déployez-le sur Cloudflare Pages.
+
+## Installation
+
+Ajoutez l'adaptateur Cloudflare pour activer SSR dans votre projet Astro avec la commande `astro add` suivante. Cela installera l'adaptateur et apportera les changements appropriés à votre fichier `astro.config.mjs` en une seule étape.
```

Comment:
I don't know if my suggestion (with the `le SSR` instead of just `SSR`) is better or not. But for me, the `suivante` word is too much.
```suggestion
Ajoutez l'adaptateur Cloudflare pour activer le SSR dans votre projet Astro avec la commande `astro add`. Cela installera l'adaptateur et apportera les changements appropriés à votre fichier `astro.config.mjs` en une seule étape.
```